platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/sync/protocol/SyncProtocol.java

```diff
@@ -107,7 +108,7 @@ public class SyncProtocol implements Protocol {
      * @param intakeIsTooFull     returns true if the intake queue is too full to continue syncing, false otherwise
      * @param sleepAfterSync      the amount of time to sleep after a sync
      * @param syncMetrics         metrics tracking syncing
-     * @param statusNexus         provides the current platform status
+     * @param platformStatusSupplier         provides the current platform status
```

Comment:
indentation is out of alignment with other parameters.

Reply:
fixed